mx.sulong/mx_sulong.py

```diff
@@ -1115,6 +1115,22 @@ def checkNoHttp(args=None):
                 exit(-1)
             line_number += 1
 
+def dos2unix(args=None):
+    """convert file from DOS to UNIX file format. Functionality equivalent to UNIX command 'dos2unix'"""
+    for fileName in args:
+        if not os.path.isfile(fileName):
```

Comment:
Could we throw an hard error here (e.g., by using `exit`)? If we pass an invalid path to the function, then we have an error in the logic of our mx script and probably always want to quit.